style.css

```diff
@@ -1,8 +1,135 @@
-footer {
-  background-color: blue;
+html {
+  width: 1440px;
+  height: 3473px;
+}
+
+header {
+  margin-bottom: 90px;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  font-family: 'gilroy', sans-serif;
+  font-weight: 800;
+  font-size: 20px;
+  font-style: normal;
+  line-height: 25px;
+  border-bottom: 3px solid #309;
+  border-top: 3px solid #309;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: #585858;
+  cursor: pointer;
+  margin-bottom: 41px;
+  margin-right: 42px;
+  margin-top: 44px;
+  line-height: 25px;
+}
+
+.nav__link:hover {
+  color: red;
+}
+
+.nav__link_button {
+  font-family: 'montserrat', sans-serif;
+  font-weight: 800;
+  font-style: normal;
+  color: #fff;
+  line-height: 24px;
+  width: 180px;
+  height: 50px;
+  background-color: #309;
+  border-radius: 50px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 30px;
+  margin-right: 130px;
+  margin-top: 30px;
+}
+
+.preview {
+  display: block;
+  background-image: url(images/background.png);
+  background-size: auto;
+  height: 481px;
+}
+
+.preview__title {
+  display: block;
+  margin-top: 151px;
+  margin-bottom: 8px;
+  margin-right: 130px;
+  line-height: 49px;
+  font-size: 40px;
+  text-align: right;
+  float: right;
+  color: #309;
+  width: 566px;
+  height: 98px;
+  font-weight: 900;
+}
+
+.preview__slogan {
+  display: block;
+  color: #FFF;
```

Comment:
Codacy found an issue: [Expected "#FFF" to be "#fff" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Azez3/pullRequest?prid=6273092)